### DISCUSSION TITLE
Add Dataverse TS File Command missing

### DISCUSSION THREAD

**Discussion by SkippingAdi** [Power-Maverick/DataverseDevTools-VSCode#164] [Q&A] [answered]
Hi all
I just installed Version 2.2.0 of this tool in Visual Studio Code, followed your instructions about connecting to the environment and creating a new project. Anyways what kind I select, TypeScript or WebPack, I couldn't see the "Add Dataverse TS File" command in the context menu.
Is this one of the integration issues you have currently? What's the workaround?
Aedu

**Comment by Power-Maverick** [comment, marked as answer]
Once you setup the project using "TypeScript" or "Webpack" then you should be able to use "Add Dataverse TS File" option. TypeScript project will initiate a regular TS project with proper tsconfig file whereas Webpack will create a TS project but with webpack configurations so you can get the benefits of tree shaking, etc.

If you arent familiar with webpack; I'd recommend you to start looking into it but in meantime you can start with a TypeScript project.

**Comment by kkazala** [comment]
I have the same problem. No matter which type of a project I create, I don't see the "Add Dataverse TS/JS file" option, only "Add new RESX" file. 
![image](https://github.com/user-attachments/assets/c4c2b7e7-5ce8-45f2-a981-dbcc7c8d1cfa)

I'm also on v2.2.0. 
Help? =)

**Comment by Power-Maverick** [comment]
Sometimes due to how VSCode works you migh have to reopen the folder or the workspace
